credsweeper/utils/util.py

```diff
@@ -270,6 +272,53 @@ def read_data(path: str) -> Optional[bytes]:
             logger.error(f"Unexpected Error: Can not read '{path}'. Error message: '{exc}'")
         return None
 
+    @staticmethod
+    def get_xml_data(file_path: str) -> Tuple[Optional[List[str]], Optional[List[int]]]:
+        """Read xml data and return List of str.
+
+        Try to read the xml data and return formatted string.
+
+        Args:
+            file_path: path of xml file
+
+        Return:
+            List of formatted string(f"{root.tag} : {root.text}")
+
+        """
+        lines = []
+        line_nums = []
+        try:
+            with open(file_path, "r") as f:
+                xml_lines = f.readlines()
+            tree = etree.fromstringlist(xml_lines)
```

Comment:
```suggestion
            tree = etree.parse(file_path)
```
May we use etree's file reading? I've not found xml in UTF-16 but it seems possible.

Reply:
Actually I used `etree.parse()`, but there is no `sourceline` attribute. It is needed to recognize exact line number of xml file.
So I change it to use `etree.fromstring()`.